.nvidia-ci.yml

```diff
@@ -149,6 +151,64 @@ raise-publishing-merge-request:
     - cnt-ngc-publish render --project-name "gpu-operator" --versions-file "build-info-${CI_PIPELINE_ID}.txt" --output gpu-operator.yaml
     - cnt-ngc-publish merge-request --files "gpu-operator.yaml"
 
+# Raise an MR to publish the image to NGC
+release:ngc-image-publish:
+  extends:
+    - .publish-images
+    - .set-publish-images-values
+  rules:
+    - if: $CI_COMMIT_TAG
+  needs:
+    - job: release:ngc-gpu-operator
+
+# Create a dummy MR that exercises the publishing logic
+release:dummy-image-publish:
+  extends:
+    - .publish-images
+    - .set-publish-images-values
+  needs:
+    - job: release:staging-gpu-operator
```

Comment:
```suggestion
    - job: update-nspect-staging
```

Reply:
It seems like this hasn't been addressed.